core/src/test/java/ai/timefold/solver/core/impl/testdata/domain/inheritance/singlevarinterface/baseannotated/childannotatedaddvar/TestdataSolution.java

```diff
@@ -0,0 +1,86 @@
+package ai.timefold.solver.core.impl.testdata.domain.inheritance.singlevarinterface.baseannotated.childannotatedaddvar;
```

Comment:
Example of shortening:

```suggestion
package ai.timefold.solver.core.impl.testdata.domain.inheritance.singlevarinterface.baseannotated.childtoo.addvar;
```